invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/frontpage.html

```diff
@@ -49,9 +49,10 @@ <h3 class="section-title">{{ _("Customize") }}</h3>
           </div>
           <h3 class="section-title">{{ _("Deploy it!") }}</h3>
           <div class="command-line">
-            {% trans deployment_link="https://inveniordm.docs.cern.ch/deployment/" %}Use the Helm charts to deploy your
-            InvenioRDM instance. You can use the Helm chart to <a href="{{ deployment_link }}">deploy in
-              OpenShift</a>.{% endtrans %}
+            {% trans deployment_link="https://inveniordm.docs.cern.ch/deployment/" %}
+              Use the Helm charts to deploy your InvenioRDM instance. You can use the Helm chart to 
+              <a href="{{ deployment_link }}" target="_blank">deploy in OpenShift</a>.
```

Comment:
The documentation for k8s is only on GitHub, but is linked from the docs page. Shall we mention it?

```suggestion
              <a href="{{ deployment_link }}" target="_blank">deploy in OpenShift or a bare Kubernetes cluster</a>.
```